.github/workflows/e2e_pr.yml

```diff
@@ -44,13 +44,11 @@ jobs:
           DATABASE_URL: postgres://postgres:postgres@localhost:5432/blurts
       - name: Store Playwright's Version
         run: |
-          # The regular expression matches everything containing
-          # `<number>.<number>.<number>`, and returns only that bit:
-          PLAYWRIGHT_VERSION=$(npm ls @playwright/test | grep @playwright | sed --regexp-extended 's/.*([0-9]+\.[0-9]+\.[0-9]+).*/\1/') | head -1
-          echo "Playwright's Version: $PLAYWRIGHT_VERSION"
+          # Get the current Playwright version listed in package.json
+          PLAYWRIGHT_VERSION=$(npx playwright --version | sed 's/Version //')
```

Comment:
If you’re lazy, like me, and don’t care about a `:`, like me, you could drop the `sed` and delete the “Version:” string below and just echo “Playwright $PLAYWRIGHT_VERSION” since the box playwright—version includes that “Version” prefix.